server/views/components/actions-card/template.njk

```diff
@@ -1,8 +1,37 @@
-<div class="actions-list">
-  <ul class="govuk-list actions-list-list">
-    <span class="govuk-heading-m actions-list-header">Actions</span>
-    {% for action in actions %}
-      <li><a class="govuk-link" {% if action.dataQa %} data-qa="{{ action.dataQa }}" {% endif %}href="{{ action.href }}">{{ action.title }}</a></li>
-    {% endfor %}
-  </ul>
-</div>
+{% from "govuk/macros/attributes.njk" import govukAttributes %}
+
+{% if params.actions | length %}
+
+  {#- Set classes for this component #}
+  {%- set classNames = "govuk-summary-card actions-card" -%}
+  {%- if params.classes %}
+    {% set classNames = classNames + " " + params.classes %}
+  {% endif %}
+
+  {#- Define component attributes #}
+  {%- set componentAttributes %} class="{{ classNames }}" {{- govukAttributes(params.attributes) -}} {% if params.id %} id="{{ params.id }}"{% endif %}{% endset %}
+
+  <div {{ componentAttributes | safe }}>
+    <div class="govuk-summary-card__title-wrapper govuk-!-padding-bottom-2">
```

Comment:
The component is based on a GDS Summary Card, so uses the same markup and GDS css classes